src/main/java/com/cybozu/kintone/client/module/parser/Parser.java

```diff
@@ -5,8 +5,8 @@
 import com.google.gson.JsonElement;
 
 public class Parser {
-	private static final Gson gson = new Gson();
-	public Object parseObject(Object obj) throws KintoneAPIException {
+	protected final Gson gson = new Gson();
```

Comment:
I think following code needs to be fix like this.
https://github.com/kintone/kintone-java-sdk/blob/v0.6.0_update_parser/src/main/java/com/cybozu/kintone/client/module/parser/FormFieldParser.java#L56

Reply:
I've updated the class `FormFieldParser` in commit [79e4d66](https://github.com/kintone/kintone-java-sdk/pull/47/commits/79e4d66690734b7a603551483c850fd2ee98538f)